packages/big-design/src/components/Stepper/styled.tsx

```diff
@@ -0,0 +1,112 @@
+import { theme as defaultTheme } from '@bigcommerce/big-design-theme';
+import styled, { css } from 'styled-components';
+
+import { DashProps, LightProps, TextProps } from './Stepper';
+
+export const StyledStepper = styled.div`
+  display: grid;
```

Comment:
We should be extending from our built in component rather than defining our own display properties, where possible.
```suggestion
export const StyledStepper = styled(Grid)`
```

That way you can use built in helpers:
```tsx
<StyledStepper marginBottom="xLarge" />
```